### PR TITLE
Update fury/minim

### PR DESCRIPTION
**Pull request by kylef**
This pull request updates fury/minim to the latest version(s) and updates the necessary code to be compatible. See in-line comments below.

### CODE REVIEW

File: src/compile.coffee [hunk 1]
```diff
@@ -73,7 +73,7 @@ findRelevantTransactions = (mediaType, parseResult) ->
         relevantTransactions.push(httpTransaction) unless httpTransaction.attributes.example is example
       else
         relevantTransactions.push(httpTransaction)
-      example = httpTransaction.attributes.example
+      example = httpTransaction.attributes?.example
```
Comment (kylef) [inline src/compile.coffee]: Attributes won't be included in elements if it is empty anymore. I did see some checks for attributes but some we're missing that I have added.

File: src/parse.coffee [hunk 2]
```diff
@@ -33,7 +33,7 @@ parse = (source, callback) ->
       err = new Error(err.message)
 
     if apiElements
-      apiElements = apiElements.toRefract()
+      apiElements = fury.minim.toRefract(apiElements)
```
Comment (kylef) [inline src/parse.coffee]: You can get hold of the minim namespace via `fury.minim` and toRefract is namespace not element now. (Serialisation is not coupled in the element models).

**Comment by honzajavorek** [comment]
One nitpick, I'm not sure about the "chore" commit classification. I think upgrading libraries is either breaking (not this case, I believe) or fixing/adding features (?). If none of that applies, I'd still vote rather for the "refactor:" prefix in this particular case. But it's just a nitpick which doesn't affect changelog or releases, so I'm merging anyway.

**Comment by kylef** [comment]
@honzajavorek You are right, I guess it is a bug fix https://github.com/apiaryio/fury-adapter-swagger/blob/master/CHANGELOG.md#bug-fixes

**Comment by honzajavorek** [comment]
Next time then :)

**Pull request merged**